paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentMethodsUI.kt

```diff
@@ -49,7 +55,9 @@ internal fun PaymentMethodsUI(
     onItemSelectedListener: (SupportedPaymentMethod) -> Unit,
     modifier: Modifier = Modifier
 ) {
+    val context = LocalContext.current.applicationContext
     val state = rememberLazyListState()
+    val imageLoader = remember { StripeImageLoader(context) }
```

Comment:
Do we want to cache these images on device at all?

Reply:
The image loader has a default memory/disk cache as implementation details.

Reply:
Oh, right!